.github/workflows/build.yml

```diff
@@ -36,6 +36,16 @@ jobs:
       # We need a virtual X11 for the Eclipse unit tests
         run: |
           xvfb-run -a mvn -B test
```

Comment:
You should probably have `package` instead of `test` to have the dist.zip built, to avoid:

>   ls: /github/workspace/infinitest-intellij/target/infinitest-intellij-*-dist.zip: No such file or directory

Reply:
Thanks for noticing! 
In retrospect this is quite obvious but at the same time I was completely off track on that one... Thanks!